pkgs/development/python-modules/pymc3/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchurl
+, buildPythonPackage
+, Theano
+, gfortran
+, openblas
+, numpy
+, scipy
+, matplotlib
+, h5py
+, pandas
+, patsy
+, tqdm
+, joblib
+}:
+
+buildPythonPackage rec {
+  pname = "pymc3";
+  version = "3.2";
+
+  propagatedBuildInputs = [
+    Theano
+    gfortran
```

Comment:
does it need to propagate gfortran and openblas?

Reply:
Definitely not needed. I found some existing nix expression where they were listed and did not think about it honestly.